src/pages/default/ParentPageLayout.tsx

```diff
@@ -0,0 +1,27 @@
+import { ColorTheme } from "@/theme/colors";
+import { Layouts } from "@/theme/layouts";
+import React from "react";
+import { SafeAreaView, StyleSheet, View } from "react-native";
+
+export const ParentPageLayout = ({ children }) => {
```

Comment:
In my experience abstractions like this don't scale well on mobile and tend to cause more problems than they solve